qcodes/instrument_drivers/Keysight/keysight_b220x.py

```diff
@@ -0,0 +1,181 @@
+from qcodes import VisaInstrument
+from qcodes.utils.validators import MultiType, Ints, Enum, Lists
+
```

Comment:
Notebook:
> Note: Everytime set_unused_inputs is called, the previous setting is overwritten!

typo: `unused_inputs.set(...)`

Reply:
I will change that. (I renamed the function somewhere and forgot to update the description in the notebook.)

Reply:
changed to 
`Note: Everytime unused_inputs(<list of unused inputs>) is called, the previous setting is overwritten!`